packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -127,11 +131,11 @@ export default class Editor implements IEditor {
      * Dispose this editor, dispose all plugins and custom data
      */
     public dispose(): void {
-        for (let i = this.core.plugins.length - 1; i >= 0; i--) {
-            this.core.plugins[i].dispose();
+        for (let i = this.getCore().plugins.length - 1; i >= 0; i--) {

```

Comment:
For those cases where core is used multiple places in one function, you can save it to a local var first, then no need to core getCore() many times:

```ts
const core = this.getCore();
...
```

Same for other functions.